### DISCUSSION TITLE
Issue Validating Gnosis Personal Sign returned from Wallet Connect V2

### DISCUSSION THREAD

**Discussion by heyskylark** [WalletConnect/walletconnect-monorepo#2919] [Developer Support]
Hey,

When trying to initiate a personal_sign through wallet connect v2 to a gnosis safe, I am having some issues validating the signature. Originally our code use to look for `0x` returned from the personal_sign request to know that it was a contract based signature that needed to be verified using EIP-1271. It appears that after upgrading to the latest version of web3-react and using wallet connect v2 (from v1) the personal_sign request returns a large signature: 

```
0x7dee429af2b2b3610c93b81e0e01dfa638fbf98be234962819a5d02d0b6a4a796e560d82c765a4a916407893d14dd8c81e56c2caf45dadadb170e44d46c9a3ee1b62ca0fa6809c834cef5382c8f3067f8af231cf9f1b0da54cf68e2afe99a84e5630fa6c32d09c256644056b568c4a2a21d83d1475967506f03a22a33f29344da41b
```

I also tried directly validating the signature using `isValidSignature` on the contract with no success.

I was wondering, after upgrading to v2, were there any changes made with how we should handle personal_sign signatures from smart contracts?

For the wallet connect interaction, I'm currently using:
@web3-react/core: 8.2.0
@web3-react/walletconnect-v2: 8.3.6

**Comment by glitch-txs** [comment]
hey is this still an issue?
